src/python/grpcio_tests/tests/unit/_metadata_code_details_test.py

```diff
@@ -302,26 +302,33 @@ def testSuccessfulStreamStream(self):
         self.assertEqual(_DETAILS, response_iterator_call.details())
 
     def testAbortedUnaryUnary(self):
-        self._servicer.set_code(_NON_OK_CODE)
-        self._servicer.set_details(_DETAILS)
-        self._servicer.set_abort_call()
-
-        with self.assertRaises(grpc.RpcError) as exception_context:
-            self._unary_unary.with_call(object(), metadata=_CLIENT_METADATA)
-
-        self.assertTrue(
-            test_common.metadata_transmitted(
-                _CLIENT_METADATA, self._servicer.received_client_metadata()))
-        self.assertTrue(
-            test_common.metadata_transmitted(
-                _SERVER_INITIAL_METADATA,
-                exception_context.exception.initial_metadata()))
-        self.assertTrue(
-            test_common.metadata_transmitted(
-                _SERVER_TRAILING_METADATA,
-                exception_context.exception.trailing_metadata()))
-        self.assertIs(_NON_OK_CODE, exception_context.exception.code())
-        self.assertEqual(_DETAILS, exception_context.exception.details())
+        # calling abort should always fail an RPC, even for "invalid" codes
+        server_codes = [_NON_OK_CODE, 3, grpc.StatusCode.OK]
```

Comment:
Please make this and the next line tuples instead of lists

Reply:
Done. I've also moved them as module-level constants, like `_NON_OK_CODE`, since they are now referenced by 4 tests.